lib/js/dom.nim

```diff
@@ -40,8 +40,8 @@ type
     onsubmit*: proc (event: Event) {.nimcall.}
     onunload*: proc (event: Event) {.nimcall.}
 
-  # https://developer.mozilla.org/en-US/docs/Web/Events
   DomEvent* {.pure.} = enum
+    ## `https://developer.mozilla.org/en-US/docs/Web/Events`_
```

Comment:
You **have** to run nim doc (and inspect html) to check whether output looks good at least until you're familiar enough with nim doc.
The link format you're using is still wrong and gives broken links; see my original comment where i gave you a working format https://github.com/nim-lang/Nim/pull/13483#discussion_r390724719

However I must admit RST format is hard to read/write/remember and you can now use the much more standard markdown format for links:

```
    ## see [docs](https://developer.mozilla.org/en-US/docs/Web/Events)
```

Reply:
I made the links be in the format of "see <docs>" as requested.
![image](https://user-images.githubusercontent.com/125948/76424244-362d3300-6365-11ea-8fde-66d669230bff.png)
I made sure the links click. Sorry very rusty on my RST formatting rules.